### ISSUE TITLE
Self-hosted battles

### ISSUE THREAD

**Issue by Jazcash** [beyond-all-reason/bar-lobby#53]
Add as an option in host battle dialog, but default to using dedicated servers

**Comment by p2004a** [comment]
I don't believe we right now plan in any way to support purely user hosted battles, at least it's something that is very very far off, so I'm closing for now.
